### PR TITLE
AAP-12616 add instructions for updating execution environment images …

### PR DESCRIPTION
**Pull request by hherbly**
…to Installation Guide.

See [AAP-12616](https://issues.redhat.com/browse/AAP-12616) for more info.

### CODE REVIEW

File: downstream/modules/platform/proc-update-ee-image-locations.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+// assembly-platform-whats-next.adoc
+
+:_content-type: PROCEDURE
+
+[id="updating-ee-image-locations_{context}"]
+
+= Updating execution environment image locations
+
+[role="_abstract"]
+If your {PrivateHubName} was installed separately, you may need to update your execution environment image locations to point to your {PrivateHubName}. Use this procedure to update your execution environment image locations.
```
Comment (dcdacosta) [inline downstream/modules/platform/proc-update-ee-image-locations.adoc:11]: IBM style says not to use may but rather can or might. 

"To indicate possibility, use "might". To indicate permission, use "may".
In most cases, use "can" or "might", not "may"."

**Review by dcdacosta — APPROVED** [review]
One minor comment, otherwise, LGTM!

**Review by dcdacosta — APPROVED** [review]
LGTM!

File: downstream/modules/platform/proc-update-ee-image-locations.adoc (hunk 1 above)
Comment (hherbly) [reply downstream/modules/platform/proc-update-ee-image-locations.adoc]: Adding for posterity: I updated the language to "can" as in "you can update your ee image locations" to eliminate any ambiguity that would be introduced by "might". Thanks Donna!

**Pull request merged**